test/apps/rails4/app/controllers/another_controller.rb

```diff
@@ -14,6 +19,10 @@ def use_bad_thing
     User.where(@bad_thing)
   end
 
+  def set_something
+    @something = 1
+  end
+
   def also_use_bad_thing
     `#{@bad_thing}`
```

Comment:
This is now being reported on line 25 [in this test](https://github.com/presidentbeef/brakeman/pull/746/files#diff-e1a71bca43733699848bdc597bda2091R958). This is do to the multiline array literal now taking 3 lines instead of 1. If you add more lines to [this array](https://github.com/presidentbeef/brakeman/pull/746/files#diff-58c2bbb1455b2cc5a517d6860e3c9397R11) the reported line numbers will grow by one for each new line you add to the array literal.